rust/agama-cli/src/logs.rs

```diff
@@ -22,10 +23,25 @@ pub enum LogsCommands {
     List,
 }
 
+// main entry point called from agama CLI main loop
 pub async fn run(subcommand: LogsCommands) -> anyhow::Result<()> {
     match subcommand {
-        LogsCommands::Store { verbose } => Ok(store(verbose)?),
-        LogsCommands::List => Err(anyhow::anyhow!("Not implemented")),
+        LogsCommands::Store { verbose } => {
+            // feed internal options structure by what was received from user
+            // for now we always use / add defaults if any
+            let mut options = LogOptions::new();
```

Comment:
I would suggest to implement [Default](https://doc.rust-lang.org/std/default/trait.Default.html) for `LogOptions` and change this code to:

```rust
let options = LogOptions { verbose, ..Default::default() }
```

And the same below (see the following comment).

Reply:
ah ... another chapter ... nice tip ;-)